docker-compose.yml

```
@@ -29,7 +29,8 @@ services:
     container_name: relay_nakama # Do not change this without changing the e2e-nakama makescript or e2e test will fail
     platform: linux/amd64
     build:
-      context: ./relay/nakama
+      context: ./
+      dockerfile: relay/nakama/Dockerfile
       target: nakama # Build the amd64 version of the image
```

yamllint

[warning] too few spaces before comment

(comments)